content/applications/services/helpdesk/advanced/after_sales.rst

```diff
@@ -2,34 +2,34 @@
 After-Sales services
 ====================
 
-*After-Sales* services can be configured in the *Helpdesk* application for individual *teams*. Once
+*After-Sales* services can be configured in the *Helpdesk* application for individual teams. Once
 enabled, users can issue refunds, process returns, generate coupons, and/or schedule repair and
 field service interventions directly from a ticket.
 
 Set up the after-sales services
 ===============================
 
-Start by enabling the after-sales services on specific *Helpdesk* team(s), by going to
-:menuselection:`Helpdesk --> Configuration --> Teams` and selecting which teams(s) these services
-should be active on. Then, scroll to the :guilabel:`After-Sales` section on the team's settings
-page, and choose which of the following options to enable:
+Start by enabling the after-sales services on specific a *Helpdesk* team, by going to
+:menuselection:`Helpdesk app --> Configuration --> Helpdesk Teams` and selecting which teams these
+services should be active on. Then, scroll to the :guilabel:`After-Sales` section on the team's
```

Comment:
```suggestion
:menuselection:`Helpdesk app --> Configuration --> Helpdesk Teams` and click on the team the
services should be applied to. Then, scroll to the :guilabel:`After-Sales` section on the team's
```

I thought I could select multiple teams at first, then enable something. I just wanted to make it clear that you select one at a time, and repeat for all teams.